app/components/subcase/description-panel/edit.js

```diff
@@ -151,32 +151,20 @@ export default class SubcaseDescriptionEdit extends Component {
     if (agendaitemTypeChanged) {
       await this.updateNewsletterAfterRemarkChange();
       await this.updateDecisionReport(propertiesToSetOnAgendaitem.number);
-      await this.recalculateAllAgendaitemNumbersOnAgenda()
+      if (this.agendaItemType.uri === CONSTANTS.AGENDA_ITEM_TYPES.NOTA) {
+        const agenda = await this.store.queryOne('agenda', {
+          'filter[agendaitems][agenda-activity][subcase][:id:]': this.args.subcase.id,
+          'filter[:has-no:next-version]': 't',
+        });
+        await this.agendaService.reorderAgenda(agenda);
+      }
```

Comment:
`reorderAgendaitemsOnAgenda` also checks if any decision reports need to be regenerated, only the agendaitems that were reordered need it. That is missing with this change.